tests/src/test/java/org/eclipse/hono/tests/IntegrationTestSupport.java

```diff
@@ -224,6 +227,7 @@ public String getRandomTenantId() {
      * @see #deleteObjects(TestContext)
      */
     public String getRandomDeviceId(final String tenantId) {
+        tenantsToDelete.add(tenantId);
```

Comment:
we do not add the tenant to the set of tenants to delete in order to be able to safely use the `DEFAULT_TENANT` (which shouldn't be deleted)

Reply:
oh i see..